charts/rstudio-pm/README.md.gotmpl

```diff
@@ -7,6 +7,18 @@
 
 {{ template "rstudio.install" . }}
 
+## Required Configuration
+
+This chart requires the following in order to function:
+
+* A license key, license file, or address of a running license server. See the `license` configuration below.
+* A Kubernetes [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) that contains the data directory for RSPM.
```

Comment:
I think we may need to clean up our terminology here (although I admit my usage of these terms is not very clean).

The way that the chart usually works is to set up a PVC with an appropriate storage class, which _generates_ the PV "behind the scenes." 

https://stackoverflow.com/a/48956790/6570011

In fact, I believe sometimes a user will "have access to create a PVC," but "not create a PV," because the storage class does that.